<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-CKSETLD" failures="1" tests="4" name="Test Listener" time="16.011" errors="0" timestamp="16 Apr 2020 20:38:59 GMT">
  <testcase classname="testNG.TestCases" name="login" time="5.08"/>
  <testcase classname="testNG.TestCases" name="testToFail" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at testNG.TestCases.testToFail(TestCases.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- testToFail -->
  <testcase classname="testNG.TestDependency" name="openBrowser" time="6.503"/>
  <testcase classname="testNG.TestDependency" name="loginFacebook" time="6.777"/>
</testsuite> <!-- Test Listener -->
